opencti-platform/opencti-graphql/tests/03-streams/00-Raw/raw-test.js

```diff
@@ -36,7 +36,7 @@ describe('Raw streams tests', () => {
       }
       // 02 - CHECK UPDATE EVENTS
       const updateEvents = events.filter((e) => e.type === EVENT_TYPE_UPDATE);
-      expect(updateEvents.length).toBe(106);
+      expect(updateEvents.length).toBe(108);
```

Comment:
Can you explains why?

Reply:
Test relationsAdd ()multiple add relation) on stix-core and stix-sighting
2 new updates